third_party/webrender/webrender/Cargo.toml

```diff
@@ -9,7 +9,7 @@ build = "build.rs"
 edition = "2018"
 
 [features]
-default = ["freetype-lib"]
+default = ["freetype-lib", "no_static_freetype"]
```

Comment:
This feature should probably not be added unconditionally, but added via a platform-specific dependency in the compositor's Cargo.toml.